README.md

```diff
@@ -100,6 +100,9 @@ This is list of known compatible USB hubs:
 | Raspberry Pi       | 3B+        ([see below](#raspberry-pi-3b))           | 4     | 2.0 |`0424:2514`| 2018    |      |
 | Raspberry Pi       | 4B         ([see below](#raspberry-pi-4b))           | 4     | 3.0 |`2109:3431`| 2019    |      |
 | Renesas            | uPD720202 PCIe USB 3.0 host controller               | 2     | 3.0 |           | 2013    |      |
+| Revolution Pi      | Core 3 / S / SE                                      | 2     | 2.0 |`0424:9514`| 2017    |      |
```

Comment:
* Please replace manufacturer column from `Revolution Pi` to `Kunbus` - official company name manufacturing Revolution Pi.
* Please replace model from `Core ...` to `RevPi Core ...`. Searching online shows that most places know this as `RevPi`, not `Revolution Pi`
* Please sort list alphabetically, e.g. `RevPi Connect` should be sorted before `RevPi Core`

Reply:
Fixed it